min-turnup/phase2/Kconfig

```diff
@@ -3,14 +3,18 @@ menu "Phase 2: Node Bootstrapping"
 
 config phase2.docker_registry
 	string "docker registry"
-	default "gcr.io/google-containers"
+	default "docker.io/colemickens"
 	help
 	  The docker registry to pull cluster components from.
 
 config phase2.kubernetes_version
 	string "kubernetes version"
-	default "v1.2.4"
+	default "v1.3.0-azure"
```

Comment:
Yes, but the Azure bring-up will not work as-is without using my branch with the `cloudprovider`. That's one of the caveats I called out that makes this a bit harder to handle.

If it were possible to use flannel (without requiring the docker bootstrap) then we could potentially support Azure + min_turnup without requiring the cloudprovider. I'm not keen on doing that immediately since I still need to get the cloudprovider reviewed/merged first.

I'm open to suggestions on how to handle this.